index.bs

```diff
@@ -5748,7 +5748,7 @@ Note: this extension may be implemented for [=authenticators=] that do not use [
         ::  One or two inputs on which to evaluate PRF. Not all [=authenticators=] support evaluating the PRFs during credential creation so outputs may, or may not, be provided. If not, then an [=assertion=] is needed in order to obtain the outputs.
 
         :   <dfn>evalByCredential</dfn>
-        ::  A record mapping [=base64url encoding|base64url encoded=] [=credential IDs=] to PRF inputs to evaluate for that credential. Only applicable during [=assertions=] when {{PublicKeyCredentialRequestOptions/allowCredentials}} is not empty.
+        ::  {{evalByCredential}} is a record mapping [=base64url encoding|base64url encoded=] [=credential IDs=] to PRF inputs to evaluate for that credential. Only applicable during [=assertions=] when {{PublicKeyCredentialRequestOptions/allowCredentials}} is not empty.
```

Comment:
It doesn't look like this reference is necessary, since there are other refs to `evalByCredential` below. Is it? I think the text reads better without this change.

Reply:
adding `{{evalByCredential}}` to prose eliminated a bikeshed `LINK ERROR` -- it was apparently complaining about there being no explicit links to that webIDL item.  i say we leave it in (am trying to eliminate errors to make echinda (sp?) happy).

Reply:
yeah, that is a lame attempt at eliminating the remaining LINK ERROR, and it didn't work so i've yanked it.